faq-site-client/src/components/global/NavDrawer.vue

```diff
@@ -70,6 +72,21 @@
                 }
             }
         },
+        watch: {
+            $route(to: Route, from: Route) {
+                if (to.fullPath.includes(Routes.TOPIC)) {
+                    this.activeTopicHash = [+(to.params as any).hash]; //Perhaps not use this later on, but doing this thru the store
+                }
+            },
+            activeTopicHash(hash: number[]) {
+                if (hash.length !== 0) {
```

Comment:
Can be simplified to `if (hash.length)`

Reply:
Now it also checks if the types are the same